examples/vault-iam-auth/auth-signature-scripts/install-dependencies.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+function has_apt_get {
+  [[ -n "$(command -v apt-get)" ]]
+}
```

Comment:
I know this is a tiny bash script, but given that this script is targeted at Ubuntu, it may make sense to use the bash-commons functions we have for this: https://github.com/gruntwork-io/bash-commons/blob/master/modules/bash-commons/src/os.sh#L19-L24